roles/api-server/defaults/main/main.yaml

```diff
@@ -27,10 +27,11 @@ api_server_masterdata_api_port: 8443
 api_server_metal_api_url: http://metal-api.metal-control-plane.svc.cluster.local:8080
 api_server_metal_api_hmac: metal-admin
 
-api_server_auditing_url: http://meili-meilisearch.{{ metal_stack_cloud_namespace }}.svc.cluster.local:7700
+api_server_auditing_url: http://auditing-meili-meilisearch.metal-control-plane.svc.cluster.local:7700
```

Comment:
maybe now we need to create a `auditing` namespace and put meilisearch there ? WDYT ?

Reply:
Good point. Introducing an `auditing` namespace would probably be cleaner as we now know that we might introduce alternative backends in the future. 

But I am not sure if it's worth the effort _at the moment_ to extract auditing into a new role in the mini-lab.

Reply:
We can keep this in the mini-lab as it is, but extend the meilisearch role to have a configurable namespace. For other environments we can then split up and deploy meilisearch into a different namespace.

Reply:
And merged. :)